ros_buildfarm/config/plot_config.py

```diff
@@ -42,6 +42,9 @@ def __init__(self, name, data):
             'stackedBar', 'stackedBar3d', 'waterfall')
 
         self.y_axis_exclude_zero = bool(data.get('y_axis_exclude_zero', False))
+        
```

Comment:
Please address the CI failure: https://travis-ci.org/ros-infrastructure/ros_buildfarm/jobs/627639835

Reply:
Done